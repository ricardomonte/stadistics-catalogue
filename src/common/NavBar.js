import React from 'react';
import { Link, NavLink } from 'react-router-dom';
import GameStyles from '../styles/NavBar.module.css';

const NavBar = () => {
  const isActive = {
    color: '#4addeb',
  };
  return (
    <header className={GameStyles.containerHeader}>
      <div className={GameStyles.containerLogo}>
        <Link to="/" exact className={GameStyles.logo}>
          fgtg
        </Link>
      </div>
      <nav className={GameStyles.contNav}>
        <NavLink
          activeStyle={isActive}
          to="/"
          className={GameStyles.linkHoAb}
          exact
        >
          Home
        </NavLink>
        {' | '}
        <NavLink
          activeStyle={isActive}
          to="/about"
          className={GameStyles.linkHoAb}
        >
          About
        </NavLink>
      </nav>
    </header>
  );
};

export default NavBar;
