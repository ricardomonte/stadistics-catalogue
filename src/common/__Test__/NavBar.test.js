import React from 'react';
import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import NavBar from '../NavBar';

test('Conten from Home matches snapshot', () => {
  const content = render(
    <BrowserRouter>
      <NavBar />
    </BrowserRouter>,
  );
  expect(content.container).toMatchSnapshot();
  expect(screen.getByText('fgtg')).toBeInTheDocument();
});
