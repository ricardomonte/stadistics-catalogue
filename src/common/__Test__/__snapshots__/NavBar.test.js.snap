// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Conten from Home matches snapshot 1`] = `
<div>
  <header
    class="containerHeader"
  >
    <div
      class="containerLogo"
    >
      <a
        class="logo"
        href="/"
      >
        fgtg
      </a>
    </div>
    <nav
      class="contNav"
    >
      <a
        aria-current="page"
        class="linkHoAb active"
        href="/"
        style="color: rgb(74, 221, 235);"
      >
        Home
      </a>
       | 
      <a
        class="linkHoAb"
        href="/about"
      >
        About
      </a>
    </nav>
  </header>
</div>
`;
